includes/class-wc-subscriptions-email-notifications.php

```diff
@@ -20,23 +20,30 @@ class WC_Subscriptions_Email_Notifications {
 	 */
 	public static $switch_setting_string = '_customer_notifications_enabled';
 
+	/**
+	 * Init.
+	 */
 	public static function init() {
 
 		add_action( 'woocommerce_email_classes', __CLASS__ . '::add_emails', 10, 1 );
 
 		add_action( 'woocommerce_init', __CLASS__ . '::hook_notification_emails' );
 
+		// Add notification actions to the admin edit subscriptions page.
 		add_filter( 'woocommerce_order_actions', [ __CLASS__, 'add_notification_actions' ], 10, 1 );
 
 		// Trigger actions from Edit order screen.
 		add_action( 'woocommerce_order_action_wcs_customer_notification_free_trial_expiration', [ __CLASS__, 'forward_action' ], 10, 1 );
 		add_action( 'woocommerce_order_action_wcs_customer_notification_subscription_expiration', [ __CLASS__, 'forward_action' ], 10, 1 );
 		add_action( 'woocommerce_order_action_wcs_customer_notification_renewal', [ __CLASS__, 'forward_action' ], 10, 1 );
 
+		// Add settings UI.
 		add_filter( 'woocommerce_subscription_settings', [ __CLASS__, 'add_settings' ], 20 );
 
 		add_action( 'update_option_' . WC_Subscriptions_Admin::$option_prefix . self::$offset_setting_string, [ 'WC_Subscriptions_Email_Notifications', 'set_notification_settings_update_time' ] );
 		add_action( 'update_option_' . WC_Subscriptions_Admin::$option_prefix . self::$switch_setting_string, [ 'WC_Subscriptions_Email_Notifications', 'set_notification_settings_update_time' ] );
+		add_action( 'add_option_' . WC_Subscriptions_Admin::$option_prefix . self::$offset_setting_string, [ 'WC_Subscriptions_Email_Notifications', 'set_notification_settings_update_time' ] );
```

Comment:
This addresses the issue where the callback is missed on the first write for existing installations.